specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NHOperationsList.json

```diff
@@ -11,237 +11,211 @@
             "display": {
               "provider": "Microsoft Azure Notification Hub",
               "resource": "Microsoft Azure Notification Hub",
-              "operation": "Registers the NotificationHubs Provider",
-              "description": "Registers the subscription for the NotifciationHubs resource provider and enables the creation of Namespaces and NotificationHubs"
```

Comment:
I think, these descriptions shouldn't be removed.

Reply:
It's not part of the "display" https://github.com/Azure/azure-rest-api-specs/blob/master/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json#L1522-L1540
I think they probably got add because of the VSCode bug, where for the example files, it's applying the Swagger and Example schema